src/chat.cpp

```diff
@@ -594,7 +618,30 @@ bool CHAT_Input( event_t *pEvent )
 			}
 			else if ( pEvent->data1 == '\t' )
 			{
-				g_ChatBuffer.TabComplete();
+				if (( g_ulChatMode == CHATMODE_PRIVATE_SEND ) && ( SERVER_CountPlayers( false ) >= 2 ))
```

Comment:
This is also breaking now when `tempPlayer == MAXPLAYERS `, so there is no need anymore to check `SERVER_CountPlayers( false ) >= 2`, right?

Reply:
`SERVER_CountPlayers( false ) >= 2` is still needed, since there's no reason to execute the loop and try finding another client who we can privately chat with if none exist. However, the server (MAXPLAYERS) is always a valid receiver, so in this case, we don't really need to check if `tempPlayer` is a valid receiver using `chat_IsPlayerValidReceiver( tempPlayer )`. I hope this makes sense.

Reply:
Good point.